### PR TITLE
add usage example in readme, update membrane site links

### PR DESCRIPTION
**Pull request by WojciechBarczynski**

### CODE REVIEW

**Review by jerzywilczek — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -16,6 +16,48 @@ Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
...
+    links = [
+      link(:file_src)
+      |> to(:parser)
+      |> to(:sdl)
+    ]
+    
+    {{:ok, spec: %ParentSpec{children: children, links: links}}, %{}}
+  end
+end
+```
```
[the remark this answers is not in the document]
Comment (WojciechBarczynski) [reply README.md]: done

File: .gitignore [hunk 2]
```diff
@@ -2,6 +2,7 @@ tmp
 compile_commands.json
 .elixir_ls
 .gdb_history
+priv/
```
Comment (jerzywilczek) [inline .gitignore:5]: Can you check whether this is necessary?

Comment (jerzywilczek) [reply .gitignore:5]: Bc from what I understand this is not in our standard `.gitignore`

**Review by jerzywilczek — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (WojciechBarczynski) [reply .gitignore]: I had updated dialyzer config and removed it

**Review by jerzywilczek — APPROVED** [review]

**Pull request merged**